index.node.js

```diff
@@ -0,0 +1,5 @@
+import { convertToMarkdown } from './lib/convert.js';
+import jsdom from 'jsdom';
+
+global.DOMParser = new jsdom.JSDOM().window.DOMParser;
+global.convertToMarkdown = convertToMarkdown;
```

Comment:
Was there a reason *not* to use `retype-parse` here instead, which has the same interface as `rehype-dom-parse`, rather than adding extra steps and opportunities for problems (e.g. script injection issues)?